### PR TITLE
 Update the 'Node Help' for the pi-sense-hat node

### PR DESCRIPTION
**Pull request by juggledad**
Expanded the explanation for using multiple commands.
Provided examples for Rotate, Flip and Display.
Updated README.md

- [x] add to the documentation of the node
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Proposed changes

Adds examples  adn enhanses the 'multiple commands' explaination.

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [x] For non-bugfix PRs, I have discussed this change on the mailing list/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

File: hardware/sensehat/README.md [hunk 1]
```diff
@@ -71,7 +71,9 @@ Joystick events are sent when the Sense HAT joystick is interacted with. The
 This node sends commands to the 8x8 LED display on the Sense HAT.
 
 Commands are sent to the node in `msg.payload`. Multiple commands can
-be sent in a single message by separating them with newline (\n) characters.
+be sent in a single message by separating them with newline (\n) characters. 
+You must use a function node or a change node(jsonata expression) when
```
Comment (dceejay) [inline hardware/sensehat/README.md]: can we have a space between node and (

File: hardware/sensehat/README.md [hunk 2]
```diff
@@ -71,7 +71,9 @@ Joystick events are sent when the Sense HAT joystick is interacted with. The
 This node sends commands to the 8x8 LED display on the Sense HAT.
 
 Commands are sent to the node in `msg.payload`. Multiple commands can
-be sent in a single message by separating them with newline (\n) characters.
+be sent in a single message by separating them with newline (\n) characters. 
+You must use a function node or a change node(jsonata expression) when 
+using the newline (\n) character to have multiple commands
```
Comment (dceejay) [inline hardware/sensehat/README.md]: maybe ?...  to create a payload containing multiple commands.

File: hardware/sensehat/README.md [hunk 3]
```diff
@@ -103,12 +105,16 @@ Format: `R<angle>`
 
 `angle` must be 0, 90, 180 or 270.
 
+Example: R180
```
Comment (dceejay) [inline hardware/sensehat/README.md]: add the backticks \` to make it a code block \`R180\`

File: hardware/sensehat/README.md [hunk 4]
```diff
@@ -103,12 +105,16 @@ Format: `R<angle>`
...
+
 #### Flip the screen
 
-Format: `R<axis>`
+Format: `F<axis>`
 
 `axis` must be either `H` or `V` to flip on the horizontal or vertical axis respectively.
 
+Example: FV
```
Comment (dceejay) [inline hardware/sensehat/README.md]: add backticks

File: hardware/sensehat/sensehat.html [hunk 5]
```diff
@@ -73,7 +73,9 @@
 <p>Raspberry Pi Sense HAT output node.</p>
 <p>This node sends commands to the 8x8 LED display on the Sense HAT.</p>
 <p>Commands are sent to the node in <code>msg.payload</code>. Multiple commands can
-be sent in a single message by separating them with newline (\n) characters.<p>
+be sent in a single message by separating them with newline (\n) characters. You must 
+use a function node or a change node(jsonata expression) when using the newline (\n)
```
Comment (dceejay) [inline hardware/sensehat/sensehat.html]: as per README connects

File: hardware/sensehat/sensehat.html [hunk 6]
```diff
@@ -100,11 +102,13 @@
 <p><b>Rotate the screen</b></p>
 <p>Format: <code>R&lt;angle&gt;</code></p>
 <p><code>angle</code> must be 0, 90, 180 or 270.</p>
+<p>Example: R180</p>
```
Comment (dceejay) [inline hardware/sensehat/sensehat.html]: add \<code\> --- ie       \<code\>R180\<\/code\>

File: hardware/sensehat/sensehat.html [hunk 7]
```diff
@@ -100,11 +102,13 @@
...
 
 <p><b>Flip the screen</b></p>
 <p>Format: <code>F&lt;axis&gt;</code></p>
 <p><code>axis</code> must be either <code>H</code> or <code>V</code> to flip on
 the horizontal or vertical axis respectively.</p>
+<p>Example: FV</p>
```
Comment (dceejay) [inline hardware/sensehat/sensehat.html]: add \<code\> tags

File: hardware/sensehat/sensehat.html [hunk 8]
```diff
@@ -121,6 +125,7 @@
 <p><b>Set the screen brightness</b></p>
 <p>Format: <code>D&lt;level&gt;</code></p>
 <p><code>level</code> must be 0 (low) or 1 (high).</p>
+<p>Example: D1</p>
```
Comment (dceejay) [inline hardware/sensehat/sensehat.html]: add \<code\> tags

File: hardware/sensehat/README.md (hunk 2 above)
Comment (juggledad) [reply hardware/sensehat/README.md]: sure

File: hardware/sensehat/README.md (hunk 1 above)
Comment (juggledad) [reply hardware/sensehat/README.md]: sure

**Pull request merged**